qiskit/visualization/pulse_v2/drawing_objects.py

```diff
@@ -70,7 +71,7 @@ class ElementaryData(ABC):
     __hash__ = None
 
     def __init__(self,
-                 data_type: str,
+                 data_type: Union[str, Enum],
```

Comment:
Are these any Enum, or are they one of the enums in `types.py`? I think you could use, for example, `types.DynamicString` as the type

Reply:
I assume end-user sets a custom data type. `str` is the simplest way to define custom type. basically this can be anything that is hashable.

Reply:
You're specifically adding `Enum`, though. Why add it?